src/Firebase.js

```diff
@@ -0,0 +1,24 @@
+import app from "firebase/app";
+
+// Add the Firebase products that you want to use
+require("firebase/auth");
+require("firebase/firestore");
+// my web app's Firebase configuration
+var firebaseConfig = {
+    apiKey: process.env.apiKey,
+    authDomain: process.env.authDomain,
+    databaseURL: process.env.databaseURL,
+    projectId: process.env.projectId,
+    storageBucket: process.env.storageBucket,
+    messagingSenderId: process.env.messagingSenderId,
+    appId: process.env.appId
+  };
+
+  // Initialize Firebase
+  class Firebase{
+    constructor() {
+        app.initializeApp(firebaseConfig);
```

Comment:
@magicmarie you should check whether there is no other Firebase app instance before initializing another app.
COnsider the code below
```if (!firebase.apps.length) {
  firebase.initializeApp(config);
}```